Symfony/CS/Tests/Tokenizer/TokensTest.php

```diff
@@ -111,4 +111,76 @@ function (array $matches) {
             ),
         );
     }
+
+    /**
+     * @dataProvider provideFindSequence
+     */
+    public function testFindSequence($source, $expected, array $params)
+    {
+        $tokens = Tokens::fromCode($source);
+
+        if ($expected instanceof \Exception) {
+            $this->setExpectedException(get_class($expected), $expected->getMessage());
+        }
```

Comment:
You have 2 very different things with ``$expected``, and ``$this->assertSame($expected, ...)`` does not make sense in the case of an exception. So this should be split in 2 different tests with separate data providers